crates/ruff_python_formatter/src/pattern/pattern_match_star.rs

```diff
@@ -1,19 +1,22 @@
-use ruff_formatter::{write, Buffer, FormatResult};
+use ruff_formatter::{prelude::text, write, Buffer, FormatResult};
 use ruff_python_ast::PatternMatchStar;
 
-use crate::{not_yet_implemented_custom_text, FormatNodeRule, PyFormatter};
+use crate::comments::dangling_comments;
+use crate::AsFormat;
+use crate::{FormatNodeRule, PyFormatter};
 
 #[derive(Default)]
 pub struct FormatPatternMatchStar;
 
 impl FormatNodeRule<PatternMatchStar> for FormatPatternMatchStar {
     fn fmt_fields(&self, item: &PatternMatchStar, f: &mut PyFormatter) -> FormatResult<()> {
-        write!(
-            f,
-            [not_yet_implemented_custom_text(
-                "*NOT_YET_IMPLEMENTED_PatternMatchStar",
-                item
-            )]
-        )
+        let comments = f.context().comments().clone();
+        let dangling = comments.dangling_comments(item);
```

Comment:
Should comments be handled in `PatternMatchSequence`?

Reply:
It depends on the comments. What's the code snipped that requires the dangling comment handling?

Reply:
@MichaReiser 

For example:

```python
x = [1, 2, 3, 4]

match x:
    case [
        1,
        *
        # comment here
        rest,
    ]:
        print(rest)
```